src/ext/local-vars.js

```diff
@@ -0,0 +1,36 @@
+htmx.defineExtension('local-vars', {
+    onEvent: function (name, evt) {
+        if (!localStorage) {
+            console.log("Warning: browser does not support local storage.")
+        }
+        elt = evt.detail.elt;
+        if (name === "configRequest.htmx") {
+            if (elt.hasAttribute("hx-include-local-vars") || elt.hasAttribute("data-hx-include-local-vars") ) {
```

Comment:
I would prefer a non-hx prefix, something like `local-vars` and `data-local-vars` so these aren't confused with a standard attribute

Reply:
ok

Reply:
Are you okay with include-local-vars and  data-include-local-vars?  If so, I think we are done :-).